src/main/java/MathOperations.java

```diff
@@ -0,0 +1,37 @@
+public class MathOperations {
+    private double a;
+    private String operator;
+    private double b;
+
+    public MathOperations(double a, String operator, double b) {
+        this.a = a;
+        this.operator = operator;
+        this.b = b;
+    }
+
+    double getResult() {
+        return switch (operator) {
+            case "+" -> a + b;
+            case "-" -> a - b;
+            case "*" -> a * b;
+            case "/" -> a / b;
+            default -> throw new IllegalStateException("Unexpected value");
+        };
+    }
+
+    public double getA() {
+        return a;
+    }
+
+    public String getOperator() {
+        return operator;
+    }
+
+    public double getB() {
+        return b;
+    }
```

Comment:
Jeżeli przy wyświetlaniu i zapisie do pliku korzystasz tylko z metody `getOperationWithResult`, to gettery możesz usunąć.